stubs/openpyxl/openpyxl/reader/excel.pyi

```diff
@@ -19,8 +28,10 @@ class ExcelReader:
     def read_workbook(self) -> None: ...
     def read_properties(self) -> None: ...
     def read_theme(self) -> None: ...
-    def read_chartsheet(self, sheet, rel) -> None: ...
+    def read_chartsheet(self, sheet: ChildSheet, rel: Relationship) -> None: ...
     def read_worksheets(self) -> None: ...
     def read(self) -> None: ...
 
-def load_workbook(filename, read_only: bool = ..., keep_vba=..., data_only: bool = ..., keep_links: bool = ...): ...
+def load_workbook(
```

Comment:
Conflict:
This, and `ExcelReader.__init__` is ultimately passed to `ZipFile` in `_validate_archive`.
`ZipFile.__init__` is typed as `ZipFile(file: StrPath | IO[bytes], ...)`

Even in read mode it still needs a `name` variable, `seek` and `tell` methods.

Reply:
I'd be okay with switching it to IO[bytes]`. We could probably come up with a more precise protocol, but for now consistency with the stdlib `zipfile` stub is probably the most important thing